packages/strapi-plugin-i18n/oas.yml

```diff
@@ -70,6 +70,39 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/locale'
+  /i18n/content-manager/actions/get-non-localized-fields:
+    post:
+      tags:
+        - Content-Manager
+      description: Return the non localized fields of a related entity in order to pre-fill the creation form of a related entity.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/getNonLocalizedFieldsInput'
+      responses:
+        200:
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  nonLocalizedFields:
+                    type: object
+                  localizations:
+                    description: All related entities id and locale
+                    type: array
+                    items:
+                      type: object
+                      properties:
+                        id:
+                          oneOf:
```

Comment:
use $ref: '#/components/schemas/id' ?

Reply:
yes. Btw id doesn't seem to work well, we see "one of" but not of what :p I checked and the syntax seems good so it may come from the theme. It's the same in the existing doc of the CM too

![Capture d’écran 2021-03-24 à 14 41 57](https://user-images.githubusercontent.com/11708220/112320207-18501900-8caf-11eb-8a54-3048fd2d0c82.png)

Reply:
Yeah I also noticed this in other places. We shoud look into a fix or switch back to swagger. at some point if needed